database/src/service/response.rs

```diff
@@ -0,0 +1,53 @@
+//! Read/write `Response`'s from the database.
+//!
+//! TODO: could add `strum` derives.
+
+//---------------------------------------------------------------------------------------------------- Import
+
+//---------------------------------------------------------------------------------------------------- Constants
+
+//---------------------------------------------------------------------------------------------------- Response
+#[derive(Debug)]
+/// Either a read or write response.
+///
+/// TODO: not sure if we actually need this.
+pub enum Response {
```

Comment:
We do need this, as it would be good to have the read and write services return the same thing.

I think this could be changed though as the only thing a write op should return is `Ok` so instead of having 3 enums just add `Ok` to the read response and make that the `Response` enum